torchgeo/datasets/idtrees.py

```diff
@@ -595,27 +594,12 @@ def plot_las(self, index: int, colormap: Optional[str] = None) -> Any:
         points: "np.typing.NDArray[np.int_]" = np.stack(
             [las.x, las.y, las.z], axis=0
         ).transpose((1, 0))
+        point_cloud = pyvista.PolyData(points)  # type: ignore[attr-defined]
 
-        if colormap:
-            if hasattr(mpl, "colormaps"):
-                cm = mpl.colormaps[colormap]
-            else:
-                cm = plt.cm.get_cmap(colormap)
-            norm = plt.Normalize()
-            colors = cm(norm(points[:, 2]))[:, :3]
-        else:
-            # Some point cloud files have no color->points mapping
-            if hasattr(las, "red"):
-                colors = np.stack([las.red, las.green, las.blue], axis=0)
-                colors = colors.transpose((1, 0)) / 65535
-            # Default to no colormap if no colors exist in las file
-            else:
-                colors = np.zeros_like(points)
-
-        pcd = open3d.geometry.PointCloud()
-        pcd.points = open3d.utility.Vector3dVector(points)
-        pcd.colors = open3d.utility.Vector3dVector(colors)
-        vis = open3d.visualization.Visualizer()
-        vis.create_window()
-        vis.add_geometry(pcd)
-        return vis
+        # Some point cloud files have no color->points mapping
+        if hasattr(las, "red"):
+            colors = np.stack([las.red, las.green, las.blue], axis=0)
+            colors = colors.transpose((1, 0)) / 65535
```

Comment:
I know this code was already here, but where does 65535 come from?

Reply:
the RGB values per point for whatever reason are uint16 so they need to be normalized to the range [0, 1] so they can be used as a colormap. (I didn't find any documentation why they are between [0, 65535]

Reply:
In that case I would suggest:
```suggestion
            colors = colors.transpose((1, 0)) / np.iinfo(np.uint16).max
```
so that there aren't any magic numbers.